src/components/DashboardEditor/editorUtils.jsx

```diff
@@ -206,6 +280,44 @@ export const timeRangeToJSON = {
   },
 };
 
+export const validThresholdIcons = [
```

Comment:
Could these be moved to `src/utils/bundledIcons.js`? The aim there is to have a centralized spot where we import icons for consumption via name through the dashboard/cards